src/cr/cube/stripe/cubemeasure.py

```diff
@@ -210,6 +277,32 @@ def unweighted_counts(self):
         return self._unweighted_counts[:, 0]
 
 
+class _NumArrUnweightedCubeCounts(_BaseUnweightedCubeCounts):
+    """Unweighted-counts cube-measure for a non-MR stripe."""
+
+    @lazyproperty
+    def bases(self):
+        """1D np.int64 ndarray of table-proportion denonimator (base) for each row.
+
+        Each row in a CAT stripe has the same base (the table-base).
+        """
+        return self._unweighted_counts
+
+    @lazyproperty
+    def pruning_base(self):
+        """1D np.int64 ndarray of unweighted-N for each matrix row.
+
+        Because this matrix has no MR dimension, this is simply the unweighted count for
+        each row.
```

Comment:
I notice you're leaving a lot of unrevised copy-paste errors like these that relate to the place you copied them from but not this. I recommend you make updating the docstring the first order of business when you copy a "template" from somewhere else, so you're less likely to forget.